.editorconfig

```diff
@@ -0,0 +1,14 @@
+# EditorConfig is awesome: http://EditorConfig.org
+
+root = true
+
+[*]
+end_of_line = lf
+insert_final_newline = false
+trim_trailing_whitespace = false
+
+[*.{js}]
+insert_final_newline = false
+trim_trailing_whitespace = false
+indent_style = tab
+indent_size = 2
```

Comment:
This Project also uses an Indent Size of 4. This setting would go against what is already there

Reply:
Thanks corrected.